Sources/PrimerSDK/Classes/User Interface/Root/PrimerCardFormViewController.swift

```diff
@@ -78,15 +104,35 @@ class PrimerCardFormViewController: PrimerFormViewController {
             verticalStackView.addArrangedSubview(saveCardSwitchContainerStackView)
             saveCardSwitchContainerStackView.isHidden = true
         }
+    }
+    
+    private func configureZipCodeFieldRow() {
+        zipCodeFieldRow.addArrangedSubview(formPaymentMethodTokenizationViewModel.zipCodeContainerView)
+        zipCodeFieldRow.addArrangedSubview(UIView())
+        verticalStackView.addArrangedSubview(zipCodeFieldRow)
+    }
+    
+    private func onConfigurationFetched() {
+        let zipView = formPaymentMethodTokenizationViewModel.zipCodeContainerView
+        let isZipCodeViewHidden: Bool = !zipCodeFieldRow.arrangedSubviews.contains(zipView)
```

Comment:
Why not move this logic in the view model? I mean that `formPaymentMethodTokenizationViewModel.zipCodeContainerView` can return `nil` if this logic is `true`

You can also do that in the `viewDidLoad` on this part:
```
if (formPaymentMethodTokenizationViewModel.requireZipCode) {
            configureZipCodeFieldRow()
        }
```